ci/generate_azure_pipelines_matrices.py

```diff
@@ -158,28 +158,28 @@ def __init__(self, platform, build_jobs):
             subarchives="qtbase qttools qt icu",
         ),
         BuildJob(
-            "src", "6.1.0", "linux", "desktop", "gcc_64", "gcc_64", subarchives="qt"
+            "install-src", "6.1.0", "linux", "desktop", "gcc_64", "gcc_64", subarchives="qt"
         ),
         BuildJob(
-            "doc", "6.1.0", "linux", "desktop", "gcc_64", "gcc_64", subarchives="qtdoc"
+            "install-doc", "6.1.0", "linux", "desktop", "gcc_64", "gcc_64", subarchives="qtdoc"
         ),
         # test for list commands
-        BuildJob('list', '5.15.2', 'linux', 'desktop', '', '', spec="<6", list_options={
+        BuildJob('list-qt', '5.15.2', 'linux', 'desktop', '', '', spec="<6", list_options={
             'HAS_EXTENSIONS': "True",
         }),
-        BuildJob('list', '6.1.0', 'linux', 'android', '', '', spec=">6.0,<6.1.1", list_options={
+        BuildJob('list-qt', '6.1.0', 'linux', 'android', '', '', spec=">6.0,<6.1.1", list_options={
             'HAS_EXTENSIONS': "True",
             'USE_EXTENSION': "armv7",
         }),
         # tests run on linux but query data about other platforms
-        BuildJob('list', '5.14.1', 'mac', 'ios', '', '', spec="<=5.14.1", list_options={}),
-        BuildJob('list', '5.13.1', 'windows', 'winrt', '', '', spec=">5.13.0,<5.13.2", list_options={}),
+        BuildJob('list-qt', '5.14.1', 'mac', 'ios', '', '', spec="<=5.14.1", list_options={}),
+        BuildJob('list-qt', '5.13.1', 'windows', 'winrt', '', '', spec=">5.13.0,<5.13.2", list_options={}),
```

Comment:
This change appears to have disabled all of the `aqt list-qt` and `aqt list-tool` tests. All of those tests are within this if statement:
https://github.com/miurahr/aqtinstall/blob/97c0837610f159f39030a0a9c29a4a7fa3eadd0e/ci/steps.yml#L67

If you check the CI logs, you'll see that the build job was run, but it didn't run any tests:

https://dev.azure.com/miurahr/github/_build/results?buildId=4454&view=logs&jobId=3b077b59-19b3-5c3c-d9ea-4f2d359714ca&j=3b077b59-19b3-5c3c-d9ea-4f2d359714ca&t=fcf4a360-d878-547e-2092-53786c55870d